package.json

```diff
@@ -92,10 +92,15 @@
     "nodemon": "^1.19.4",
     "postcss": "^7.0.27",
     "prettier-eslint": "^9.0.1",
-    "prettier-eslint-cli": "^5.0.0"
+    "prettier-eslint-cli": "^5.0.0",
+    "typescript": "^3.8.3"
   },
   "engines": {
     "node": "12",
     "npm": "6.13"
-  }
+  },
+  "browsers": [
+    "> 1%",
```

Comment:
> Un doute sur le % utilisé ici suite aux [Best Practices](https://github.com/browserslist/browserslist/blob/master/README.md#best-practices) de la documentation de browserslist qui dit : 
```
If you want to change the default set of browsers, we recommend combining last 2 versions, not dead with a usage number like > 0.2%. This is because last n versions on its own does not add popular old versions, while only using a percentage above 0.2% will in the long run make popular browsers even more popular. We might run into a monopoly and stagnation situation, as we had with Internet Explorer 6. Please use this setting with caution.
```

Reply:
J'ai pas d'avis tranché sur la question mais je pense qu'il n'est pas impossible que des administrateurs utilisent Internet Explorer...